meshtastic/tests/test_main.py

```
@@ -4463,43 +4557,118 @@ def test_main_ota_update_succeeds_and_prints_completion(
+        "localhost",
+        "--dest",
+        MAIN_LOCAL_ADDR,
+        "--ota-update",
+        "firmware.bin",
+    ]
+    mt_config.args = cast(Any, sys.argv)
+
+    local_node = MagicMock(autospec=Node)
+    other_node = MagicMock(autospec=Node)
+
+    def _get_node(dest: object, requestChannels: bool = True, **_kwargs: object) -> Any:
```

Ruff (0.15.4)

[error] Boolean-typed positional argument in function definition

(FBT001)